db/migrate/20230920111215_create_posts.rb

```diff
@@ -1,6 +1,7 @@
 class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
+      t.integer :AuthorId
```

Comment:
- [ ] I suggest that you use in your migrations one of the helpers that Rails offers to set relationships in the DB level:

```suggestion
      t.references :author, foreign_key: { to_table: 'users' }
```

Please consider applying this change in all your migrations. Otherwise, you would have to run new migrations to create the foreign keys and establish the corresponding relationships! 🤓 🔧

Reply:
Thank you for your kind feedback, working on it @anagudelogu.